aws_lambda.py

```diff
@@ -72,7 +72,21 @@ def validate_signature(event, context):
         computed_hash = compute_hash(serialized)
 
         # 5. Grab the signature
-        signature = dest_col['data']['signature']
+        try:
+            signature = dest_col['data']['signature']
+        except KeyError:
```

Comment:
I kind of prefer being explicit about which KeyError we're looking for and expecting here using something like `if 'signature' not in dest_col['data']:`